taskt/Core/Script/Script.cs

```diff
@@ -1,180 +1,177 @@
-
-                //increment line number
-                lineNumber++;
-            }
-
-            //output to xml file
-            XmlSerializer serializer = new XmlSerializer(typeof(Script));
-            var settings = new XmlWriterSettings
-            {
-                NewLineHandling = NewLineHandling.Entitize,
-                Indent = true
-            };
-
-            //if output path was provided
-            if (scriptFilePath != "")
-            {
-                //write to file
-                System.IO.FileStream fs;
-                using (fs = System.IO.File.Create(scriptFilePath))
-                {
-                    using (XmlWriter writer = XmlWriter.Create(fs, settings))
-                    {
-                        serializer.Serialize(writer, script);
-                    }
-                }
-            }
-
-
-            return script;
-        }
-        /// <summary>
-        /// Deserializes a valid XML file back into user-defined commands
-        /// </summary>
-        public static Script DeserializeFile(string scriptFilePath)
-        {
-            XmlSerializer serializer = new XmlSerializer(typeof(Script));
-
-            System.IO.FileStream fs;
-
-            //open file stream from file
-            using (fs = new System.IO.FileStream(scriptFilePath, System.IO.FileMode.Open))
-            {
-                //read and return data
-                XmlReader reader = XmlReader.Create(fs);
-                Script deserializedData = (Script)serializer.Deserialize(reader);
-                return deserializedData;
-            }
-        }
-        /// <summary>
-        /// Deserializes an XML string into user-defined commands (server sends a string to the client)
-        /// </summary>
-        public static Script DeserializeXML(string scriptXML)
-        {
-            System.IO.StringReader reader = new System.IO.StringReader(scriptXML);
-            XmlSerializer serializer = new XmlSerializer(typeof(Script));
-            Script deserializedData = (Script)serializer.Deserialize(reader);
-            return deserializedData;
-        }
-    }
-}
+﻿//Copyright (c) 2019 Jason Bayldon
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Windows.Forms;
+using Newtonsoft.Json;
+using taskt.Core.Automation.Commands;
+
+namespace taskt.Core.Script
+{
+    using Formatting = Newtonsoft.Json.Formatting;
```

Comment:
any particular reason why you chose to put the alias in the namespace scope?

Reply:
That's a mistake hah, I'll move it up